runner/utils/driver/istio/watcher.go

```diff
@@ -0,0 +1,65 @@
+package istio
+
+import (
+	"context"
+	"time"
+
+	"github.com/spaceuptech/helpers"
+	appsv1 "k8s.io/api/apps/v1"
+	v12 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/client-go/informers"
+	"k8s.io/client-go/informers/internalinterfaces"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/cache"
+)
+
+const (
+	// ResourceDeleteEvent syncman uses this const to represent a delete event
+	ResourceDeleteEvent = "delete"
+	// ResourceUpdateEvent syncman uses this const to represent a delete event
+	ResourceUpdateEvent = "update"
+	// ResourceAddEvent syncman uses this const to represent a delete event
+	ResourceAddEvent = "add"
+)
```

Comment:
change comments accordingly & don't export constants if they are not being used externally